packages/vats/src/core/boot.js

```diff
@@ -17,6 +17,13 @@ import * as behaviors from './behaviors.js';
 import * as clientBehaviors from './client-behaviors.js';
 import * as utils from './utils.js';
 
+// Ambient types
+import './types.js';
+// eslint-disable-next-line import/export -- no named exports
+export * from '../types.js';
```

Comment:
I just realized this file is the entrypoint of the package. It's definitely a little confusing.

Would you mind introducing an `index.js` at the root which `exports *` from both this file and `./src/types.js` ?

Reply:
Good idea